CMSgov/mint-app
# Easi 2613/user account migration plan participants and providers

# EASI-2613

## Changes and Description

- Migrate user Plan Participants and Providers to use the user table
- Update postman collection

* NOTE, this needs the change in [this PR](https://github.com/CMSgov/mint-app/pull/451/files#diff-e9c3d1e1fde2fbcec1466719be5864a4493ede9e1fce08b753df1e07d8336bfb) to pass unit tests as expected *

## How to test this change
1.  Run the app. Update plan participants and providers to make sure it functions as expected.


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
